.github/workflows/create-azure-self-hosted-runners.yml

```diff
@@ -23,14 +19,24 @@ on:
         type: string
         required: false
         description: Repo to deploy the runner to. Only needed if runner_scope is set to "repo-level" (defaults to current repository)
+      deallocate_immediately:
+        type: string
+        required: true
+        description: Deallocate the runner immediately after creating it (useful for spinning up runners preemptively)
+        default: false
 
 env:
-  AMOUNT_OF_RUNNERS: ${{ github.event.inputs.amount_of_runners }}
   ACTIONS_RUNNER_SCOPE: ${{ github.event.inputs.runner_scope }}
   ACTIONS_RUNNER_ORG: "${{ github.event.inputs.runner_org || github.repository_owner }}"
   ACTIONS_RUNNER_REPO: "${{ github.event.inputs.runner_repo || github.event.repository.name }}"
+  DEALLOCATE_IMMEDIATELY: ${{ github.event.inputs.deallocate_immediately }}
   # This has to be a public URL that the VM can access after creation
-  POST_DEPLOYMENT_SCRIPT_URL: https://raw.githubusercontent.com/${{ github.repository }}/${{ github.ref_name }}/azure-self-hosted-runners/post-deployment-script.ps1
+  POST_DEPLOYMENT_SCRIPT_URL: https://raw.githubusercontent.com/${{ github.repository }}/${{ github.ref }}/azure-self-hosted-runners/post-deployment-script.ps1
+  # Note that you'll need "p" (arm64 processor) and ideally "d" (local temp disk). The number 8 stands for 8 CPU-cores.
+  # For a convenient overview of all arm64 VM types, see e.g. https://azureprice.net/?_cpuArchitecture=Arm64
+  AZURE_VM_TYPE: Standard_D8plds_v5
+  # At the time of writing, "centralindia" was the cheapest region for the VM type we're using. https://azureprice.net/vm/Standard_D8plds_v5
+  AZURE_VM_REGION: centralindia
```

Comment:
Hmm. It does [not look as if that region has the VM type we require](https://github.com/git-for-windows/git-for-windows-automation/actions/runs/3883922611/jobs/6625799742#step:7:21).

Reply:
Interesting. AzurePrice [does list it as being available in that region](https://azureprice.net/vm/Standard_D8plds_v5). Probably "eastus" would do the trick then 🤷🏼

Reply:
https://azureprice.net/content/about says that this site is not affiliated with Microsoft, and I fail to see any equivalent prices in the Azure Portal (probably because the former tries to factor in Windows license costs?)

In any case, I think I solved the riddle about West India:

![image](https://user-images.githubusercontent.com/127790/211587163-e442435a-2e7a-426a-a765-62589ff9ddf2.png)

Unfortunately, I do not find any official documentation about pricing at https://learn.microsoft.com/en-us/azure/virtual-machines/dplsv5-dpldsv5-series, at least not anything helpful (the calculator seems to either be unaware of that series or calls it by a different name).

Reply:
I went with "westus2" and started [a new workflow run](https://github.com/git-for-windows/git-for-windows-automation/actions/runs/3884683198).

Reply:
Oh, and https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=virtual-machines&regions=central-india,south-india,west-india claims that availability of these machines in India is projected for Q1 2023.

Reply:
Almost there!! 😅 🤦🏼 https://github.com/git-for-windows/git-for-windows-automation/actions/runs/3881130673/jobs/6625764356

```
 ==> Verifying source file signatures with gpg...
    xpdf-4.00.tar.gz ... FAILED (unknown public key F4825F5397271342)
==> ERROR: One or more PGP signatures could not be verified!
```

Reply:
Yep. I actually had that fixed in git-for-windows/MINGW-packages@c8ccd71286c52c2f4df17857a2e8f40830eb20fa already, but then had a brain fart when I re-ran the failed jobs, not remembering that a re-run would fail to target the newly-pushed commit.